mkdocs/docs/litmus-namespaced-scope/litmus-namespaced-crds.yaml

```diff
@@ -2271,6 +2270,8 @@ spec:
                                     type: string
                                   initialDelay:
                                     type: string
+                                  initialDelaySeconds:
```

Comment:
Are we sure, this is integer? Other that this looks good!

Reply:
yes